artifacts/index.ts

```diff
@@ -57,4 +58,5 @@ export {
 	RSITrendingTriggerContract,
 	TimeSeriesFeedContract,
 	TwoAssetLinearizedTimeSeriesFeedContract,
```

Comment:
Export the `SocialTradingManager` too?

Reply:
Weird I thought I updated those. But yes +1